application/common/models/Password.php

```diff
@@ -241,4 +247,28 @@ public function validateNotPublicPassword($attribute)
             }
         }
     }
+
+    /**
+     * @param string $attribute
+     */
+    public function validateNotBeenPwned($attribute)
```

Comment:
what is `$attribute` here?  Is it a password?

Reply:
This is a Yii2 validator, so the required parameter is the name of the attribute being validated. In this case, `$attribute == 'password'`

Reply:
gotcha, that does ring a bell, thanks.